### PR TITLE
feat: update minimum Node.js version

### PR DESCRIPTION
**Pull request by ematipico**
#### Description (required)

This PR updates the minimum Node.js version. 

 #### For Astro version: `4.6`. See astro PR [#10689](https://github.com/withastro/astro/pull/10689). -->

### CODE REVIEW

File: src/content/docs/en/install/auto.mdx [hunk 1]
```diff
@@ -13,7 +13,7 @@ Read our [step-by-step manual installation guide](/en/install/manual/) instead.
 :::
 #### Prerequisites
 
-- **Node.js** - `v18.14.1` or higher.
+- **Node.js** - `v18.17.1` or higher.
```
Comment (Princesseuh) [inline src/content/docs/en/install/auto.mdx:16]: Wonder if this should precise the full requirements, which is 18.17.1 or higher for 18, not 19, and 20.3.0 or higher for 20.

Comment (ematipico) [reply src/content/docs/en/install/auto.mdx:16]: Good point. If users are on Node.js `v19` now try to install Astro, they will get the engine warning

Comment (ematipico) [reply src/content/docs/en/install/auto.mdx]: I applied the change in [`3a32833` (#7838)](https://github.com/withastro/docs/pull/7838/commits/3a328331e8fc6791db495c94d9990496fc39de1a), I will let Sarah decide on the format of the sentence.

File: src/content/docs/en/install/auto.mdx [hunk 2]
```diff
@@ -13,7 +13,7 @@ Read our [step-by-step manual installation guide](/en/install/manual/) instead.
 :::
 #### Prerequisites
 
-- **Node.js** - `v18.14.1` or higher.
+- **Node.js** - `v18.17.1` or higher, not `v19`, and `v20.3.0` or higher.
```
Comment (sarah11918) [inline src/content/docs/en/install/auto.mdx:16]:
```suggestion
- **Node.js** - `v18.17.1` or higher, and `v20.3.0` or higher. ( `v19` is not supported.)
```

**Pull request merged**